lib/ansible/modules/utilities/logic/include_role.py

```diff
@@ -19,16 +19,16 @@
 module: include_role
 short_description: Load and execute a role
 description:
-  - Loads and executes a role as a task dynamically.
-  - This frees roles from the C(roles:) directive and allows them to be treated more as tasks.
-  - Unlike M(import_role), most keywords, including loop, with_items, and conditionals, apply to this statement.
-  - The do until loop is not supported on M(include_role).
+  - Dynamically loads and executes a specified role as a task.
+  - May be used only where Ansible tasks are allowed - inside C(pre_tasks:), C(tasks:), or C(post_tasks:) playbook objects, or as a task inside a role.
+  - Supports most keywords, including C(loop), C(with_items), and C(conditionals) (unlike M(import_role)).
```

Comment:
`import_role` supports those keywords, but for inheritance, for include it applies to the task itself, the original phrasing was more accurate